test/integration_test.py

```diff
@@ -109,6 +116,9 @@
     AlwaysTearDownTestCase,
     AzulTestCase,
 )
+from lambdas.service.app import (
```

Comment:
Not how we import the app module. Look for `load_app_module` and friends and be sure you understand why this is bad.